src/main/features/response/form/models/defendantPaymentOption.ts

```diff
@@ -2,14 +2,16 @@ import { IsDefined, IsIn } from 'class-validator'
 import { ResponseType } from 'response/form/models/responseType'
 
 export class DefendantPaymentTypeLabels {
-  static readonly INSTALMENTS: string = 'By instalments'
+  static readonly INSTALMENTS: string = 'I`ll suggest a repayment plan'
   static readonly FULL_ADMIT_BY_SPECIFIED_DATE: string = 'Full amount by a set date'
   static readonly BY_SET_DATE: string = 'By a set date'
```

Comment:
Why do we have both `FULL_ADMIT_BY_SPECIFIED_DATE` and `BY_SET_DATE`? It seems like we could use `FULL_ADMIT_BY_SPECIFIED_DATE`. @rattur please confirm and make appropriate removal.